packages/shell-evaluator/src/shell-evaluator.ts

```diff
@@ -78,12 +78,12 @@ class ShellEvaluator {
         return this.help();
       case 'enableTelemetry':
         if (this.container) {
-          this.container.toggleTelemetry(true);
+          return this.container.toggleTelemetry(true);
         }
         return;
       case 'disableTelemetry':
         if (this.container) {
-          this.container.toggleTelemetry(false);
+          return this.container.toggleTelemetry(false);
```

Comment:
this needs a return, since `this.container.toggleTelemetry()` returns a string. If we don't return, output is just `undefined`. Please let me know if not having a `return` here was intentional, and I am missing something!